.rubocop.yml

```diff
@@ -7,17 +7,11 @@ inherit_gem:
     - "config/default.yml"
 
 Metrics/LineLength:
-  Max: 120
+  Max: 80
```

Comment:
Why 80? 🤔 We all have wide screen monitors 😄

Reply:
Well personally I use 80 because I split my wide screen in 3. One for the file tree and two for source files:

![captura de tela de 2018-05-10 15-46-54](https://user-images.githubusercontent.com/1172532/39887790-7a2287fc-5469-11e8-8b58-1383529bbc58.png)

There's also github split diff, which splits the screen in two.

But I get your point. There should be no rule coming from no where.
There are some other issues with this rubocop config that should be addressed too, gonna work on them. Maybe next PR

Reply:
👍